backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/sales/ordercheckup/legacy/report.jrxml

```diff
@@ -15,6 +15,13 @@
 		<defaultValueExpression><![CDATA[".\\"]]></defaultValueExpression>
 	</parameter>
 	<parameter name="RECORD_ID" class="java.lang.Integer"/>
+	<parameter name="AD_Table_ID" class="java.lang.Integer">
+		<defaultValueExpression><![CDATA[new Integer(0)]]></defaultValueExpression>
+	</parameter>
+	<parameter name="RECORD_ID_ADJUSTED" class="java.lang.Integer" isForPrompting="false">
```

Comment:
why not simply call this parameter C_Order_ID?

Reply:
thx, adjusted